lib/cli.js

```diff
@@ -9,8 +9,19 @@ const configuration = getConfig();
 const choicesList = choices(configuration);
 const questionsList = questions(choicesList);
 
-inquirer.prompt(questionsList).then((answers) => {
-  const message = answers.moreInfo ? `${answers.editor}` : `${answers.type} ${answers.description}`;
+const prompt = inquirer.prompt(questionsList);
+
+prompt.ui.process.subscribe((answer) => {
+  if (answer.name === 'commitMessageWarning') {
+    const continueExecution = answer.answer;
+    if (!continueExecution) {
+      process.exit();
```

Comment:
Ok I thought I wasn't clear enough. That was the whole plan since https://github.com/JPeer264/node-semantic-git-commit-cli/issues/7#issuecomment-287198501